<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidgetClass</class>
 <widget class="QWidget" name="MainWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>740</width>
     <height>540</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="ImageListPage">
    <widget class="QListWidget" name="ImagelistWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>740</width>
       <height>540</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="MyImagePage">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>MyImage</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SettingPage">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="ImageListBnt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ImageList</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MyImageBnt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MyImage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SettingBnt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Role</string>
   </property>
  </widget>
  <widget class="QLabel" name="ImagePreview">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>40</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ImageNameInfo">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>300</y>
     <width>256</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>460</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetDesktop">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>460</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SetDesktop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>540</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ImageSizeInfo">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>380</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>PushButtonClicked()</slot>
 </slots>
</ui>
